CenterForOpenScience/ember-preprints
# [EOSF-656] Remove hint.css, replace usage with ember-bootstrap tooltip

## Ticket
https://openscience.atlassian.net/browse/EOSF-656

## Purpose
Prevent licensing problems and remove potential problematic tooltip usage.

## Changes
- Remove all usage and installation of hint.css (due to potential license concerns)
- Replace usages with ember-bootstrap tooltip (and do package setup so that can be used properly)
- Remove tooltips on launch page due to conflicts with custom taxonomies.

## QA Considerations
- Launch page should no longer show tooltips
- Supplementary file bar should still use tooltips
- Should show tooltip if user found on contributor add search (on submit page) is already a contributor
